bee-storage/src/sqlx.rs

```diff
@@ -60,69 +58,124 @@ use std::collections::{
     HashSet,
 };
 
-use std::rc::Rc;
+use std::{
+    rc::Rc,
+    slice,
+    vec,
+};
 
 use async_trait::async_trait;
 use futures::executor::block_on;
 use sqlx::{
+    postgres::PgQueryAs,
+    FromRow,
     PgPool,
     Row,
 };
 
 use crate::sqlx::sql_statements::*;
 
-pub const FAILED_ESTABLISHING_CONNECTION: &str = "failed to establish connection.";
-pub const CONNECTION_NOT_INITIALIZED: &str = "connection was not established and therefor is uninitialized.";
-
-#[derive(Debug, Clone)]
-pub enum SqlxBackendError {
-    ConnectionBackendError(String),
-    EnvError(std::env::VarError),
-    SqlxError(String),
-    Bincode(String),
-    UnknownError,
-    //...
+struct TransactionWrapper(pub bee_bundle::Transaction);
```

Comment:
Looks like these wrappers are not public to outside. I think you don't need to add `pub` keyword to its fields. They are visible within the same module.

Reply:
Still learning :)
fixed